src/Components/Facility/InventoryLog.tsx

```diff
@@ -146,43 +147,42 @@ export default function InventoryLog(props: any) {
           </p>
         </td>
         <td>
-          <Tooltip
-            title={
-              inventoryItem.probable_accident ? (
+          <div className="tooltip">
+            <div className="tooltip-left tooltip-text">
+              {inventoryItem.probable_accident ? (
                 <div className="text-sm leading-snug text-justify">
                   <b>Unmarks this transaction as accident</b>
                   <br />
                   This action will not affect the total stock.
                 </div>
               ) : (
-                <div className="text-sm leading-snug text-justify">
+                <div className="text-sm leading-snug text-justify ">
                   <b>Marks this transaction as accident</b>
                   <br />
                   This action will not affect the total stock. To delete the
                   transaction, create another transaction that undos the effect
                   of this, or click <i>Delete Last Entry</i>.
                 </div>
-              )
-            }
-            arrow={true}
-          >
-            <button
+              )}
+            </div>
+            <ButtonV2
               onClick={(_) => flagFacility(inventoryItem.external_id)}
               disabled={saving}
-              className="btn btn-default"
+              variant="secondary"
             >
               {inventoryItem.probable_accident ? (
                 <span className="text-primary-500">
```

Comment:
Avoid setting the color of text inside ButtonV2. ButtonV2 applies it based on the variant.

This is to keep design consistent and easy to maintain.

See how other Buttons with icons are used.